tradeexecutor/testing/ethereumtrader_uniswap_v2.py

```diff
@@ -0,0 +1,145 @@
+"""Ethereum test trading."""
```

Comment:
Too much duplicate code.

- Rearchitect this so that only one EthereumTrader class is needed.

- EthereumTrader should not care if it is trading Uniswap v2 or v3, buy() and sell() is the same regardless of the market

- Pass any models as constructor arguments if needed